git/create-a-pr.md

```diff
@@ -0,0 +1,44 @@
+# How To: Create a PR
+
+## Step 1: Ensure your repo/remotes are set up properly
+
+You can find a guide here:
+
+https://github.com/pachonk/guides/blob/master/git/fork-and-setup-remotes.md
```

Comment:
Looks like this links back to the forked repo instead of the file you're adding to this repo - will it work with a relative link instead of an absolute one?